### PR TITLE
Refactor receipt checking 

### PR DESCRIPTION
**Pull request by skogman**
- [x] Fix ethereum withdrawal receipt check

### CODE REVIEW

File: src/store/gateway/reactions.ts [hunk 1]
```diff
@@ -147,20 +147,21 @@ export function gatewayReactions(store: Store<DashboardState>) {
   async function checkIncompleteTransfers() {
 
     const plasmaGateways = PlasmaGateways.service()
+    const loomGateway = plasmaGateways.ethereumLoomGateway
+    const mainGateway = plasmaGateways.ethereumMainGateway
+    const loomReceipt = await loomGateway.withdrawalReceiptAsync(loomGateway.caller)
+    const mainReceipt = await mainGateway.withdrawalReceiptAsync(mainGateway.caller)
+    if (!loomGateway && !mainReceipt) return
```
Comment (redben) [inline src/store/gateway/reactions.ts]: Did you mean loomReceipt instead of loomGateway?

File: src/store/gateway/index.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ import { ethereumModule } from "../ethereum"
 import { TransferGatewayTokenKind } from "loom-js/dist/proto/transfer_gateway_pb"
 import { ActionContext } from "./types"
 import Axios from "axios"
+import { sleep } from "@/utils"
```
Comment (redben) [inline src/store/gateway/index.ts]: Can remove this.

File: src/store/gateway/reactions.ts (hunk 1 above)
Comment (skogman) [reply src/store/gateway/reactions.ts]: 🤦‍♂️

File: src/store/gateway/index.ts (hunk 2 above)
Comment (skogman) [reply src/store/gateway/index.ts]: good catch, removed!

**Pull request merged**